gno.land/pkg/gnoclient/client_test.go

```diff
@@ -1,100 +1,377 @@
 package gnoclient
 
 import (
-	"testing"
-
 	"github.com/gnolang/gno/gno.land/pkg/integration"
-	"github.com/gnolang/gno/gnovm/pkg/gnoenv"
-	rpcclient "github.com/gnolang/gno/tm2/pkg/bft/rpc/client"
+	abci "github.com/gnolang/gno/tm2/pkg/bft/abci/types"
+	ctypes "github.com/gnolang/gno/tm2/pkg/bft/rpc/core/types"
+	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"github.com/gnolang/gno/tm2/pkg/crypto"
 	"github.com/gnolang/gno/tm2/pkg/crypto/keys"
-	"github.com/gnolang/gno/tm2/pkg/log"
 	"github.com/gnolang/gno/tm2/pkg/std"
+	"github.com/jaekwon/testify/assert"
 	"github.com/jaekwon/testify/require"
-)
-
-func newInMemorySigner(t *testing.T, chainid string) *SignerFromKeybase {
-	t.Helper()
 
-	mmeonic := integration.DefaultAccount_Seed
-	name := integration.DefaultAccount_Name
+	"testing"
+)
 
-	kb := keys.NewInMemory()
-	_, err := kb.CreateAccount(name, mmeonic, "", "", uint32(0), uint32(0))
-	require.NoError(t, err)
+func TestClient_Render(t *testing.T) {
```

Comment:
This can be run in parallel

Reply:
[800d188](https://github.com/gnolang/gno/pull/1565/commits/800d18802726a818701bd3ec8d14d42cc145e763)